atc/metric/emitter/prometheus.go

```diff
@@ -425,14 +425,20 @@ func (emitter *PrometheusEmitter) workerContainersMetric(logger lager.Logger, ev
 		logger.Error("failed-to-find-platform-in-event", fmt.Errorf("expected platform to exist in event.Attributes"))
 		return
 	}
+	team, exists := event.Attributes["team"]
+	if !exists || team == "" {
```

Comment:
This will be a problem for those workers that don't have a team assigned to them, as in their registration/heart-beating, they'll reply with an empty string for `team`, thus making those who don't have per-team workers have any way of checking this metric.

Would it make more sense to do something like cAdvisor does with empty labels? https://github.com/google/cadvisor/issues/1951

That way we would have consistency in the label set, although I'm not entirely sure if this is the best & recommended way to go.

Wdyt?

Reply:
Thanks, I think you are right. I will ensure teamless worker also emit this metric.